configurator/configurator-impl/src/main/java/org/codice/ddf/admin/configurator/impl/AdminOperation.java

```diff
@@ -24,14 +29,36 @@
 import javax.management.MalformedObjectNameException;
 
 import org.codice.ddf.admin.configurator.ConfiguratorException;
+import org.codice.ddf.admin.configurator.Operation;
+import org.codice.ddf.admin.configurator.Result;
+import org.codice.ddf.internal.admin.configurator.opfactory.AdminOpFactory;
 import org.codice.ddf.ui.admin.api.ConfigurationAdminMBean;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  * Transactional handler for persisting bundle configuration file changes.
  */
-public class AdminOperation implements OperationBase, Operation<Void, Map<String, Object>> {
+public class AdminOperation implements Operation<Void> {
+    public static class OpFactory implements AdminOpFactory {
```

Comment:
❓ It really feels like we're making those static inner classes just to save a few files and doing so makes it difficult to come up with good names. Should we consider pulling them out to their own files and call them `*Operations`?
If not, we could name the inner class something like `Actions` maybe?